llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -509,7 +509,7 @@ class BranchCC_rri<bits<3> funct3, string opcodestr>
   let isTerminator = 1;
 }
 
-let hasSideEffects = 0, mayLoad = 1, mayStore = 0 in {
+let hasSideEffects = 0, mayLoad = 1, mayStore = 0, UseNamedOperandTable = 1 in {
```

Comment:
What is this needed for?

Reply:
'UseNamedOperandTable = 1' builds a table that's used to find instruction operand by its name in td

Reply:
I couldn't find any use of the table in your patch. Can you point me to it?